maven_iuvs/integration.py

```diff
@@ -255,8 +255,17 @@ def fit_line(myfits, wavelength,
 
         for ispa in range(n_spa):
             waves = myfits['Observation'].data['WAVELENGTH'][0, ispa]
-            DN = myfits['detector_dark_subtracted'].data[iint, ispa]
-            DN_unc = myfits['Random_dn_unc'].data[iint, ispa]
+
+            try:
```

Comment:
better to read the file identifier and select the correct treatment for l1a or l1b files...

Reply:
Fixed by 1669a94a7aa155787effe1d66b032aecf75395d3